graphos/templates/graphos/highcharts/js_treemap.html

```diff
@@ -0,0 +1,17 @@
+<script>
```

Comment:
@yashrastogi16 Any reason we need a separate js file for this? Won't BaseHighcharts js file be sufficient for treemap?